docs/TROUBLESHOOTING.md

```diff
@@ -91,3 +91,9 @@ When running eclipse with JDK 11, you might encounter the following error messag
 ## Redeployment problems
 
 If you face problems that `sormas-ui` or `sormas-rest` cannot call the backend anymore after redeploying, please follow [this instruction](DEVELOPMENT_ENVIRONMENT.md#avoid-redeployment-problems).
+
+## Malware detection triggers
```

Comment:
It would be good to put the observable behaviour that is caused by this first, so that people facing the problem can find this.

From what I have seen the behaviour is: "Update to new payara version, dev instance refuses to start" - maybe you have additional details.